Marlin/src/module/motion.cpp

```diff
@@ -1483,6 +1483,80 @@ void set_axis_not_trusted(const AxisEnum axis) {
   #endif
 }
 
+/**
+ * Move the axis back to its home_phase if set and driver is capable (TMC)
+ *
+ * Improves homing repeatability by homing to stepper coil's nearest absolute
+ * phase position. Trinamic drivers use a stepper phase table with 1024 values
+ * spanning 4 full steps with 256 positions each (ergo, 1024 positions).
+ */
+void backout_to_tmc_homing_phase(const AxisEnum axis) {
+  #ifdef TMC_HOME_PHASE
+    static const abc_ulong_t home_phase = TMC_HOME_PHASE;
```

Comment:
The feature description states to use `-1` to disable for an axis but here you store the value into an unsigned value (uint32_t). Also below you're testing if an unsigned value is below 0.
Also why specify the variable as `static`?

I'd also prefer to see size specific variable types like `int16_t` instead of `int` but that's a really minor thing.

Reply:
Hello @teemuatlut , I took all your suggestions.
Thank you for looking.